### ISSUE TITLE
Scheduled events are firing at incorrect time for one X10 module

### ISSUE THREAD

**Issue by bkenobi** [genielabs/HomeGenie#157]
I have a module on A14 that is scheduled to fire an OFF command at sunrise and 10PM per a the Scheduled Event script.  The string I use is:

(@SolarAltitude.Morning.Sunrise.End:0 22 * * *)

The result is that I see HG fire a command at 9PM and 10PM even though 9PM is never set.  I believe I may have had 9PM as the OFF time at one point, but that is no longer the case.  I have created a thread on the forum with my modules.xml file.

http://www.homegenie.it/forum/index.php?topic=848.0

**Comment by bkenobi** [comment]
I just looked at the module again to verify the settings and found that things got broken somehow.  Perhaps the reason HG was firing at the wrong time was that it changed things for me without my notice.  It currently shows the following:
![scheduled events](https://cloud.githubusercontent.com/assets/6903751/8233646/4724a606-158c-11e5-8e11-03eceba67af8.JPG)
I changed it back to the desired setting above so hopefully that setting will stick!
